lib/active_model_serializers.rb

```diff
@@ -15,6 +15,14 @@ module ActiveModelSerializers
   autoload :Adapter
   autoload :JsonPointer
   autoload :Deprecate
+  autoload :Attribute
```

Comment:
Oh didn't see https://github.com/rails-api/active_model_serializers/pull/1301/files#r42963185. It was only a note in the RFC:

> The name can be discussed in a future pull request. For example, we can rename this to Resource following this idea more info about naming in the next section